tests/test_file.py

```diff
@@ -31,7 +31,7 @@
 from .http_mock import HttpMockSequence
 import six
 
-from oauth2client import file
```

Comment:
Massive :+1:. Who had the idea to rename? Is it worth providing the old names for compatibility for a generation?